source/NVDAHelper.py

```diff
@@ -123,20 +123,31 @@ def nvdaControllerInternal_requestRegistration(uuidString):
 
 
 @WINFUNCTYPE(c_long, c_wchar_p, c_wchar_p)
-def nvdaControllerInternal_reportLiveRegion(text, level):
+def nvdaControllerInternal_reportLiveRegion(text, politeness):
 	if not config.conf["presentation"]["reportDynamicContentChanges"]:
 		return -1
 	focus = api.getFocusObject()
 	if focus.sleepMode == focus.SLEEP_FULL:
 		return -1
 	import queueHandler
 	import speech
+	from aria import AriaLivePoliteness
 	from speech.priorities import Spri
+	politenessValue = next(
+		(v for v in AriaLivePoliteness if v._name_.lower() == politeness.lower()),
+		AriaLivePoliteness.OFF
+	)

```

Comment:
I think I'd prefer to see an explicit mapping for these values. 
Or use string values in the enum so you can construct from value. Note if you do this, an unknown value raises an error.

Reply:
Using string values would require an explicit mapping for UIA, see #11596. We'd still have to make sure that the casing is right. I decided to go for integers since you can easily go from string to enum members when using _name_. Having said that, since _name_ is supposed to be private to the enum, I created a class method on the enum instead. Let's see how that fits you.

Reply:
Mm, interesting idea.
However, reading this proposal, given that this pr will introduce a third case of aria politeness values provided as strings, I don't think there's a real bias if we go for strings here, especially since the aria spec uses strings itself.
In any case, I'm convinced now that we should no longer use an IntEnum. I prefer strings over auto, though, since it follows the spec.